src/lib.rs

```diff
@@ -19,8 +19,11 @@ pub struct Aabb {
 
 type Key = (i32, i32);
 
-fn key_from_point(point: Vec2) -> Key {
-    (point.x as i32, point.y as i32)
+fn key_from_point<const TILE_SIZE: usize>(point: Vec2) -> Key {
+    dbg!((
```

Comment:
Didn't know dbg! Returned something. TiIL. Probably still don't need it though :P